test/Microsoft.NET.Build.Tests/GivenThatWeWantToDetectDeprecatedRoslynCompilerType.cs

```diff
@@ -0,0 +1,171 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable disable
+
+namespace Microsoft.NET.Build.Tests
```

Comment:
Please use file-scoped namespace
```suggestion
namespace Microsoft.NET.Build.Tests;
```